Android-Sample/app/src/main/java/com/opentok/qualitystats/sample/MainActivity.java

```diff
@@ -146,12 +145,14 @@ private void requestPermissions() {
     public void sessionConnect() {
         Log.i(LOGTAG, "Connecting session");
         if (mSession == null) {
+/*Following code is not needed anymore on sdk 2.22.2. Used to mute audio
+New Code to mute subscriber audio in function onConnected
             // Add a custom audio device before session initialization
-            CustomAudioDevice customAudioDevice = new CustomAudioDevice(
-                    MainActivity.this);
-            customAudioDevice.setRendererMute(true);
-            AudioDeviceManager.setAudioDevice(customAudioDevice);
-
+//            CustomAudioDevice customAudioDevice = new CustomAudioDevice(
```

Comment:
Looks like the CustomAudioDevice has also been removed, why has this been removed? And if its being removed these lines should be completely removed not just commended out (as the CustomAudioDevice class has been deleted)

Reply:
The customAudioDevice was being used to mute the volume. The new SDK can mute the audio via the subscriber. Left it here with comments in case main devs might need it for something. If all cool, will remove it and the file.

Reply:
If its only being used for a feature that is now in the SDK then yes let's get rid of it completely from this example. Once you have done that I will just run up the project and double-check all is good. Thank you!